### ISSUE TITLE
Implement Authenticator for None Attestation Statement Format in webauthn4j-test

### ISSUE THREAD

**Issue by majk-p** [webauthn4j/webauthn4j#851]
TL;DR I've been using `webauthn4j-test` Authenticator implementations for e2e tests. I'd like to contribute `NoneAttestationAuthenticator` that implements None Attestation (https://www.w3.org/TR/webauthn-2/#sctn-none-attestation)

**Is your feature request related to a problem? Please describe.**
E2E testing flows for Web Authentication is not easy on JVM. I managed to create e2e tests using authenticator implementation from `webauthn4j-test` package. I wanted to test None Attestation Statement Format which the library doesn't implement.

**Describe the solution you'd like**
I have what I think is a working implementation that works in my tests. I'd be happy to contribute an implementation of `NoneAttestationAuthenticator`. Let me know if you feel like it's a valuable contribution and I'd raise a PR :wink: 

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by ynojima** [comment]
Hi, contribution is always welcome, but do you really need `NoneAttestationAuthenticator`?
`ClientPlatform` replaces original `AttestationStatement` with `NoneAttestationStatement` when `attestation` option is `none`
https://github.com/webauthn4j/webauthn4j/blob/5524f89dd3e9e16de254cb4f20f07f28b9403b07/webauthn4j-test/src/main/java/com/webauthn4j/test/client/ClientPlatform.java#L91-L93
I suppose `NoneAttestationAuthenticator` is nice to have, but not necessary as long as you use `ClientPlatform`.

**Comment by majk-p** [comment]
Thank you for pointing out selenium driver! I thought about it, but in my current e2e setup I don't use selenium. I'll definitely try to iterate towards a better solution, but in case someone finds themselves in such intermittent state they might find such intermittent solution sufficient :wink: 

I'll rise a PR soon!
